pkg/neonvm/controllers/vm_controller.go

```diff
@@ -864,12 +865,45 @@ func runnerStatus(pod *corev1.Pod) runnerStatusKind {
 	case corev1.PodFailed:
 		return runnerFailed
 	case corev1.PodRunning:
-		return runnerRunning
+		return isRunnerPodReady(pod)
 	default:
 		panic(fmt.Errorf("unknown pod phase: %q", pod.Status.Phase))
 	}
 }
 
+// isRunnerPodReady returns whether the runner pod is ready respecting the readiness probe of its containers.
```

Comment:
```suggestion
// isRunnerPodReady returns whether the runner pod is ready, respecting the readiness probe of its containers.
```

Reply:
This resolved without comment or accepting the change, was that intentional?

Reply:
the same wrong revision thing